src/durableorchestrationcontext.ts

```diff
@@ -293,9 +334,28 @@ export class DurableOrchestrationContext {
      * @returns A TimerTask that completes when the durable timer expires.
      */
     public createTimer(fireAt: Date): TimerTask {
-        const newAction = new CreateTimerAction(fireAt);
-        const task = new DFTimerTask(false, newAction);
-        return task;
+        const timerAction = new CreateTimerAction(fireAt);
+        const durationUntilFire = moment.duration(moment(fireAt).diff(this.currentUtcDateTime));
+        if (this.schemaVersion >= ReplaySchema.V3) {
+            if (!this.maximumShortTimerDuration || !this.longRunningTimerIntervalDuration) {
+                throw Error(
+                    "Replay schema version >= V3 is being used, but one or more of the properties `maximumShortTimerDuration` and `longRunningTimerIntervalDuration` are not defined. This is likely an issue with the Extension."

```

Comment:
It would be good to include actionable feedback for errors like this. In this case, let's ask users to report the bug :)

```suggestion
                    "A framework-internal error was detected: replay schema version >= V3 is being used, but one or more of the properties `maximumShortTimerDuration` and `longRunningTimerIntervalDuration` are not defined. This is likely an issue with the Durable Functions Extension. Please report this bug here: https://github.com/Azure/azure-functions-durable-js/issues "

```

Reply:
Additionally, it would be good if you could log the values for `maximumShortTimerDuration` and `longRuninngTimerIntervalDuration`. It's a bit verbose, but it will help us greatly if a bug is filed.

Reply:
I could log them, but they won't be the raw JSON values that we get from the extension. Those are parsed [here](https://github.com/Azure/azure-functions-durable-js/blob/e2f32c0a65821e53bc24c53b704e1ac3811f7edb/src/durableorchestrationcontext.ts#L58) into either a `moment.Duration` object, or `undefined`, so if we hit this line, it pretty much means they were `undefined`.

Reply:
Although I guess it would help decipher which one was undefined, or if it was both

Reply:
Yup, I suggested this to figure out which one was undefined :)

Reply:
Added the logging